src/cdk/testing/test-harnesses.md

```diff
@@ -498,8 +498,8 @@ class MyPopupHarness extends ComponentHarness {
 
 #### Accessing elements outside of the component's host element
 
-There are times when a component harness might need to access elements outside of it's corresponding
-component's host element. A good example of this is components that use the
+There are times when a component harness might need to access elements outside of its corresponding
+component's host element. A good example of this are the components that use the
```

Comment:
This still sounds a bit awkward because "A good example" is singular and "the components" is plural. Maybe change the beginning of the sentence to "Some good examples of this are"

Reply:
This does indeed read better, I changed it.